proc_info.sh

```diff
@@ -14,4 +14,78 @@ function usage(){
   echo "-c                   Show command that started the process"
   echo "-f                   Show opened files by the process"
   echo "-h                   Show this help"
+  exit 0
 }
+
+#Function for -u flag
+get_owner(){
+  pid=$1
+  user_id=$(cat /proc/$pid/status | grep Uid | awk '{ print $2 }')
+  #https://unix.stackexchange.com/a/36582/468165 - Get user by Id
+  getent passwd "$user_id" | cut -d: -f1
+}
+
+#Function for -c flag
+get_command(){
+  pid=$1
+  cat /proc/$pid/comm
+}
+
+#Function for -e flag
+get_env_vars(){
+  pid=$1
+  cat /proc/$pid/environ | tr '\0' '\n'
+}
+
+#Function for -f flag
+get_opened_files(){
+  pid=$1
+  ls -l /proc/$pid/fd | less | awk '{ print $11 }'
+}
+
+while test $# -gt 0; do
+  case "$1" in
```

Comment:
Here you check first parameter and suppose that it should be flag like `-u`. In usage function it said that `PID` should go first.